docs/products/kafka/karapace/getting-started.rst

```diff
@@ -0,0 +1,12 @@
+Getting Started with Karapace
```

Comment:
We need to add `Karapace` to the capitalization exceptions (file `capitalization_headings.yml` under `.github/vale/styles`)

Reply:
Already  listed in the` capitalization_headings.yml` file